docs/cli.md

```diff
@@ -23,28 +23,9 @@ The NEAR AI CLI allows you to interact with NEAR AI's services to [create agents
     ``` bash
     git clone git@github.com:nearai/nearai.git
     cd nearai
-    pip install -e .
+    ./install.sh
```

Comment:
Instead of pointing to `install.sh` (which assumes I use `venv` to handle my environment) I would still prefer we include instructions on how to do change python version for different environment handlers (`conda`, `venv`, `pyenv`, etc)

Reply:
I wanted to keep installation consistent from the README and docs. I brought up changing to `pip install -e .` to @gabehamilton a week or two ago but he recommended sticking with `venv` as we should have devs running this in isolation instead of installing all of our libraries into their global python.

Reply:
Ok, updated this section... Please re-review @gagdiez